stdlib/SparseArrays/src/sparsevector.jl

```diff
@@ -50,6 +54,12 @@ function nonzeros(x::SparseColumnView)
     return y
 end
 
+"""
+    SparseArrays.nonzeroinds(x)
```

Comment:
I added `SparseArrays.` prefix to indicate that it is not exported.  But maybe we should export it?

Reply:
Maybe rename it to `storedindices`. See: https://github.com/JuliaLang/julia/pull/33054#discussion_r317382059